packages/atomic/src/components/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -105,12 +106,21 @@ export class AtomicBreadbox implements InitializableComponent {
     this.updateShowMoreValue(hiddenBreadcrumbs);
   }
 
+  private updateShowLessDisplay() {
+    this.show(this.showLess);
+    if (this.showLess.offsetTop === 0) {
+      this.hide(this.showLess);
```

Comment:
Would be better to not show then hide (un-necessary reflow/redraw).

Reply:
I need to show it (display=block) in order to know if it's appearing on the first row or not